apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisChart.vue

```diff
@@ -54,8 +51,6 @@ const state = reactive({
     legends: [] as Legend[],
     chartData: [] as XYChartData[],
     chart: null as XYChart | PieChart | null,
-    queryRef: null as null|Vue,
-    chartRef: null as null|HTMLElement,
```

Comment:
question: Why did you remove this refs?

Reply:
These were for the `rendered` event, but they have all been removed since the **PDF download was removed**.